samples/CSharpSamples/AssociatedTypes.cs

```diff
@@ -0,0 +1,29 @@
+﻿namespace CSharpSamples;
+
+public partial class AssociatedTypes
```

Comment:
If this project is meant to feed into testing, it might make sense to move under the `tests/` folder -- `samples/` is for showcasing demos.

Reply:
This isn't meant for executable tests, no. It's used directly by your docfx site. Instead of typing code samples in markdown files, you can type it into compiled .cs files to prove that they are and remain valid (and refactorings fix it automatically). That's why there are multiple `#region`s in this file -- the markdown references which one to inline into a particular location.

Reply:
Oh, ok. Would it make sense to move to a different folder, preferably under docs/?

Reply:
Sure. I'll move it.